### PR TITLE
Router test example and comment out the google firestore crashy code

### PR DESCRIPTION
**Pull request by jak103**
This PR provides a test example for testing echo in a unit test. 

It also comments out some test code that I left in there that will cause a crash if you don't have the Google Cloud certificate file.

### CODE REVIEW

**Comment by hhenrichsen** [comment]
Thanks for making this example! Looks good to me!

**Comment by Jaredcscott** [comment]
Thanks for making this change!

File: server/go.mod [hunk 1]
```diff
@@ -7,12 +7,11 @@ require (
 	github.com/0xAX/notificator v0.0.0-20191016112426-3962a5ea8da1 // indirect
 	github.com/codegangsta/envy v0.0.0-20141216192214-4b78388c8ce4 // indirect
 	github.com/codegangsta/gin v0.0.0-20171026143024-cafe2ce98974 // indirect
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
```
Comment (Aaron-Fine) [inline server/go.mod:10]: Why did you get rid of this dependency?

**Comment by bjsvedin** [comment]
Can't wait till i can run tests.

File: server/go.mod (hunk 1 above)
Comment (jak103) [reply server/go.mod:10]: Good question. I have no idea.

**Pull request merged**